docs/installation/INSTALLING_MODELS.md

```diff
@@ -0,0 +1,261 @@
+---
+title: Installing Models
+---
+
+# :octicons-paintbrush-16: Installing Models
+
+## Model Weight Files
+
+The model weight files ('*.ckpt') are the Stable Diffusion "secret
+sauce". They are the product of training the AI on millions of
+captioned images gathered from multiple sources.
+
+Originally there was only a single Stable Diffusion weights file,
+which many people named `model.ckpt`. Now there are dozens or more
+that have been "fine tuned" to provide particulary styles, genres, or
+other features. InvokeAI allows you to install and run multiple model
+weight files and switch between them quickly in the command-line and
+web interfaces.
+
+This manual will guide you through installing and configuring model
+weight files.
+
+## Base Models
+
+InvokeAI comes with support for a good initial set of models listed in
+the model configuration file `configs/models.yaml`. They are:
+
+| Model                   | Weight File                   |   Description                    | DOWNLOAD FROM            |
+| ----------------------  | ----------------------------- |--------------------------------- | ----------------|
+| stable-diffusion-1.5    | v1-5-pruned-emaonly.ckpt      | Most recent version of base Stable Diffusion model| https://huggingface.co/runwayml/stable-diffusion-v1-5 |
+| stable-diffusion-1.4    | sd-v1-4.ckpt                  | Previous version of base Stable Diffusion model | https://huggingface.co/CompVis/stable-diffusion-v-1-4-original |
+| inpainting-1.5          | sd-v1-5-inpainting.ckpt       | Stable Diffusion 1.5 model specialized for inpainting | https://huggingface.co/runwayml/stable-diffusion-inpainting |
+| waifu-diffusion-1.3     | model-epoch09-float32.ckpt    | Stable Diffusion 1.4 trained to produce anime images | https://huggingface.co/hakurei/waifu-diffusion-v1-3 |
+| <all models>            | vae-ft-mse-840000-ema-pruned.ckpt   | A fine-tune file add-on file that improves face generation | https://huggingface.co/stabilityai/sd-vae-ft-mse-original/ |
+
+
+Note that these files are covered by an "Ethical AI" license which
+forbids certain uses. You will need to create an account on the
+Hugging Face website and accept the license terms before you can
+access the files.
+
+The predefined configuration file for InvokeAI (located at
+`configs/models.yaml`) provides entries for each of these weights
+files. `stable-diffusion-1.5` is the default model used, and we
+strongly recommend that you install this weights file if nothing else.
+
+## Community-Contributed Models
+
+There are too many to list here and more are being contributed every
+day. [This page](https://rentry.org/sdmodels) hosts an updated list of
```

Comment:
That rentry list is really sketchy - I don't necessarily have a problem with that. I *do* think that we should also include https://huggingface.co/sd-concepts-library as a source of more... reputable... fine-tuned models

Reply:
Good feedback. I've added a `--no-interactive` switch that restores the previous behavior. It doesn't interact with user, and it also doesn't download weight files. I think github actions are doing weight file downloading separately.

Reply:
> That rentry list is really sketchy - I don't necessarily have a problem with that. I _do_ think that we should also include https://huggingface.co/sd-concepts-library as a source of more... reputable... fine-tuned models

Are we able to load .bin files? I haven't written any code to do that.

Reply:
Yes, we can (at least could a while ago) load .bin (for embeddings at least).

Reply:
I'm learning something new every day! I checked out the --embedding_path option for the first time, and sure enough I can load one of those concept .bin files.

It looks to me as though we ought to be able to load multiple .bin files. Does this not work?

Reply:
> > That rentry list is really sketchy - I don't necessarily have a problem with that. I _do_ think that we should also include https://huggingface.co/sd-concepts-library as a source of more... reputable... fine-tuned models

I've added the link to the Hugging Face concepts library. Oddly, this page doesn't load on Linux Chrome. I had to go to Firefox to view it.

Reply:
I don't know how to load both at the same time. But it'd be useful.
I tried (which understandably only uses the last one)
`python scripts/invoke.py --embedding_path learned_embeds-2.bin --embedding_path uglysonic.bin`
And this
`python scripts/invoke.py --embedding_path learned_embeds-2.bin uglysonic.bin`
But both fail.

Reply:
Internally the code seems to handle multiple embedding paths, but perhaps one will replace the previous one. I'll give it a try.